routes/login.js

```diff
@@ -24,18 +24,18 @@ router.post('/', function(req, res) {
   }
 
   var pool = req.app.get('pool');
-  pool.query('SELECT * FROM accounts WHERE accounts.username = $1::text AND accounts.password = $2::text', [signInUsername, signInPassword],
+  pool.query('SELECT * FROM accounts WHERE accounts.username = $1 AND accounts.password = $2', [signInUsername, signInPassword],
     function (err, data) {
       if (err) {
         res.send(err);
       }
       if (data.rows.length == 0) {
         res.send("Username or password is invalid");
       } else {
-        pool.query('SELECT * FROM profiles WHERE profiles.username = $1::text', [signInUsername],
```

Comment:
Is there any reason why we're removing the `::text`? I don't have it on my own branch, but I'm assuming it's some kind of flag to prevent invalid entries or code injection?

Reply:
Chelsea said actually there's no need to specify type
https://node-postgres.com/features/queries

Reply:
Gotcha